portfolio/README.md

```diff
@@ -5,4 +5,4 @@ command:
 
 ```bash
 mvn package appengine:run
-```
```

Comment:
What changed here? Different kind of newline char?

Reply:
I honestly don't know, probably something like that.